WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2599,3 +2601,26 @@ ApptEditLocationConflictColor       = @red@
 ApptEditLocationResolvedColor       = @green@
 ApptEditLocationStatusConflict      = color:@ApptEditLocationConflictColor@;
 ApptEditLocationStatusResolved      = color:@ApptEditLocationResolvedColor@;
+
+######################################
+# Appointment widget style
+######################################
+
+Appt                = z-index:10; @FixBoxModel@; @MediumRoundCorners@; overflow:hidden; opacity:0.9;
+ApptSelected        = @Appt@; z-index:11; outline: none; border:none; box-shadow: 2px 2px 2px 0 #777, 0px 0px 1px 0px #333; opacity:1;
```

Comment:
do we have color variables for #777 and #333?

Reply:
No we don't have. We do have use of this values to some property, but those are not related at all.